### DISCUSSION TITLE
Control of Axis Q1785 LE camera for storing images / screenshots of live streaming into a local directory.

### DISCUSSION THREAD

**Discussion by HPawase** [AxisCommunications/discussions#42] [General] [answered]
Hello Team,
The main task for me is to do dynamic image analysis of moving objects which are small round metal balls. I have read the Vapix documentation but unable to use it to full extent. Basically, I tried using Vapix to change camera settings and save the images but was not able to change frame per second (fps) and shutter speed via code. So, I believe it is better to use AXIS camera software directly.

I need to set the camera for continuous live streaming and get images / screenshots at regular intervals (for eg. every 15 sec, 1 min etc.) and then automatically get stored in the local directory specified by me. I do not want to use SD card and other external storages.

It should be like I get to create my own streaming profile because the camera settings are different for each application. Then, I can choose the local directory and intervals so that the images are recorded to the specified folder (for eg. /Desktop/MyFolder). Then I can use python script to access the latest image from my specified directory and use OpenCV for image processing. 

Kindly reply for further information.

**Comment by mattias-kindborg-at-work** [comment]
Hi @HPawase!

Are you running your application on the camera or on a server with a network connection to the camera?

**Reply by HPawase** [reply]
Hi
I connect the camera to the laptop and power source via a PoE connection. Then I tried to run python code which contains the IP address for the connection. Also, to change the camera settings using VAPIX. But, it becomes difficult to change shutter speed and fps of the camera and so, not sure to use in this way. 
I wanted if the AXIS device manager or AXIS camera station or any other software can do the live streaming, take screenshots and send it to a user-selected directory. And then, I can just access the directory and do image processing.

**Reply by mattias-kindborg-at-work** [reply, marked as answer]
As you are running your code on a laptop I think you are better off using the [VAPIX snapshot API](https://www.axis.com/vapix-library/subjects/t10175981/section/t10051110/display?section=t10051110-t10003575), taking a new snapshot at an intervall of your choosing.

**Reply by HPawase** [reply]
Thank you for your answer !! 
If I have any furthers issues, I will discuss with you again.
